ci/install_winpcap.bat

```diff
@@ -1,7 +1,10 @@
+:: Install WinPcap Runtime
 git submodule update --init --recursive
 git clone https://github.com/mfontanini/winpcap-installer.git
-cd winpcap-installer
-winpcap-boundary-meter-4.1.3.exe /S
-cd ..
-curl https://www.winpcap.org/install/bin/WpdPack_4_1_2.zip --output WpdPack_4_1_2.zip
-7z x .\WpdPack_4_1_2.zip -oc:\
\ No newline at end of file
+winpcap-installer\winpcap-boundary-meter-4.1.3.exe /S
+rm -rf winpcap-installer
```

Comment:
since this should run on Windows - maybe replace this with something like:
```shell
rmdir c:\test /s /q
``` 

https://superuser.com/questions/179660/how-to-recursively-delete-directory-from-command-line-in-windows

Reply:
done